radialgraph/src/main/java/com/duartbreedt/radialgraph/drawable/GraphDrawable.kt

```diff
@@ -29,11 +34,67 @@ abstract class GraphDrawable(
 
         return Paint().apply {
             strokeWidth = graphConfig.strokeWidth
-            color = state.color
             pathEffect = DashPathEffect(floatArrayOf(state.length!!, state.length!!), phase)
             style = Paint.Style.STROKE
             flags = Paint.ANTI_ALIAS_FLAG
             strokeCap = graphConfig.capStyle.paintCapStyle
+
+            if (!graphConfig.isGradientEnabled()) {
+                color = state.color
+            }
+
+            if (graphConfig.gradientType == GradientType.SWEEP) {
+                applySweepGradient(state, this, graphConfig.gradientFill)
+            }
+        }
+    }
+
+    private fun applySweepGradient(state: SectionState, paint: Paint, gradientFill: GradientFill) {
```

Comment:
Why pass `gradientFill` in as a parameter if we can simply get it off of `graphConfig` as done with `graphConfig.capStyle` below?

Reply:
I think I forgot about that. Will remove.